pkg/kubelet/kubelet_getters.go

```diff
@@ -194,6 +194,21 @@ func (kl *Kubelet) getRuntime() kubecontainer.Runtime {
 	return kl.containerRuntime
 }
 
+// getRemappedIDs returns the uid and gid from the host usernamespace that are mapped to container user-namespace
+func (kl *Kubelet) getRemappedIDs() (error, int, int) {
+	ci, err := kl.containerRuntime.GetRuntimeConfigInfo()
+	if err != nil {
+		return fmt.Errorf("Container runtime info get failed: %v", err), -1, -1
```

Comment:
to lower here

Reply:
if we cached the remappedIDs value, we could avoid recomputing it all the time.  it feels like this should be cached.